tests/e2e/steps_active_set_changes.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+
+// stepsActiveSetChanges starts a top N provider chain and causes a change in the active set
+func stepsActiveSetChanges() []Step {
+	s := []Step{
+		// === setup provider chain, consumer chain with top N = 100, and start IBC connections ===
+		{
+			Action: StartChainAction{
+				Chain: ChainID("provi"),
+				Validators: []StartChainValidator{
+					{Id: ValidatorID("alice"), Stake: 100000000, Allocation: 10000000000},
+					{Id: ValidatorID("bob"), Stake: 200000000, Allocation: 10000000000},
+					{Id: ValidatorID("carol"), Stake: 700000000, Allocation: 10000000000},
+				},
+			},
+			State: State{
+				ChainID("provi"): ChainState{
+					ValPowers: &map[ValidatorID]uint{
```

Comment:
Maybe, add a comment that max validators are set to 2 here and that's why `alice` has 0 power.